atd-etl/cris_import/cris_import.py

```diff
@@ -588,12 +652,14 @@ def align_records(map_state):
 
                 # generate some record specific SQL fragments to identify the record in larger queries
                 record_key_sql, import_key_sql = util.get_key_clauses(table_keys, output_map, table, source, map_state["import_schema"])
+                if not record_key_sql or not import_key_sql:
+                    continue
 
                 # To decide to UPDATE, we need to find a matching target record in the output table.
                 # This function returns that record as a token of existence or false if none is available
                 if util.fetch_target_record(pg, output_map, table, record_key_sql):
                     # Build 2 sets of 3 arrays of SQL fragments, one element per column which can be `join`ed together in subsequent queries.
-                    column_assignments, column_comparisons, column_aggregators, important_column_assignments, important_column_comparisons, important_column_aggregators = util.get_column_operators(target_columns, no_override_columns, source, table, output_map, map_state["import_schema"])
```

Comment:
cool so this stuff can be removed now that we are not concerned with "important columns" since the VZ layer let's us separate CRIS data from edits. As I'm getting into these details, it reminded me to ask about https://github.com/cityofaustin/atd-vz-data/pull/1240. Does that PR still fit into the code that we are trying to push into the LDM collector branch?

Reply:
Won't fib that I had a moment of fear while I tried to think about how all the branches related, but it's all good. This PR's branch actually came off of the one in 1240, and merging this would include that work entirely. I think I'll close that PR.